services/app-api/prisma/schema.prisma

```diff
@@ -577,15 +581,38 @@ enum SubmissionType {
 }
 
 model OAuthClient {
-  id            String    @id @default(uuid())
-  clientId      String    @unique
-  clientSecret  String
-  grants        String[]
-  createdAt     DateTime  @default(now())
-  updatedAt     DateTime  @default(now()) @updatedAt
-  lastUsedAt    DateTime?
-  description   String?
-  contactEmail  String?
+  id           String    @id @default(uuid())
+  clientId     String    @unique
+  clientSecret String
+  grants       String[]
+  createdAt    DateTime  @default(now())
+  updatedAt    DateTime  @default(now()) @updatedAt
+  lastUsedAt   DateTime?
+  description  String?
+  contactEmail String?
 
   @@map("oauth_clients")
 }
+
+model DocumentZipPackage {
+  id        String   @id @default(uuid())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @default(now()) @updatedAt
+  s3URL     String
+  sha256    String?
+
+  contractRevisionID String?
+  contractRevision   ContractRevisionTable? @relation(fields: [contractRevisionID], references: [id])
+
+  rateRevisionID String?
+  rateRevision   RateRevisionTable? @relation(fields: [rateRevisionID], references: [id])
+
+  documentType DocumentZipType
+}
+
+enum DocumentZipType {
+  CONTRACT_DOCUMENTS
+  CONTRACT_SUPPORTING_DOCUMENTS
```

Comment:
Are we planning to separate out the document packages? I know on the frontend, all contract documents are zipped into one package and all rates are zipped into one package. If not, should there be only 2 enum types here for all contract and all rate documents? Or we could add those 2 enums to these.

Reply:
Yeah good call, we can probably keep this the same then and I'll make this enum just `CONTRACT_DOCUMENTS` and `RATE_DOCUMENTS`